r/R/expression.R

```diff
@@ -111,7 +111,8 @@ cast_array_expression <- function(x, to_type, safe = TRUE, ...) {
 
 .unary_function_map <- list(
   "!" = "invert",
-  "is.na" = "is_null"
+  "is.na" = "is_null",
+  "is.nan" = "is_nan"
```

Comment:
It's beautiful that this is the only change needed to support this in the dplyr code.

Reply:
Yeah, absolutely! There's more testing in this PR than operational code :)